### PR TITLE
feat: added new apis

### PR DESCRIPTION
**Pull request by mfateminia**

### CODE REVIEW

File: src/services/FacilityService.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const KEYS = {
+    facilities: "facilities",
+  };
+  export function getAllFacilities() {
+    if (localStorage.getItem(KEYS.facilities) == null)
+      localStorage.setItem(KEYS.facilities, JSON.stringify([]));
+    return myPromise.then((value) => value);
+  }
```
Comment (mfateminia) [inline src/services/FacilityService.ts:8]: export const getAllFacilities = () => {
    const facilityes = localStorage.getItem(KEYS.facilities);
    if (!facilityes) {
        return [];
    }
    return facilityes;
}

Comment (mfateminia) [reply src/services/FacilityService.ts:8]: change it to async/await

File: src/services/FacilityService.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  
+  const myPromise = new Promise((resolve, reject) => {
```
Comment (mfateminia) [inline src/services/FacilityService.ts:10]: convert this to a general function that receives response of api call and resolves it in 1 sec

Comment (mfateminia) [reply src/services/FacilityService.ts:10]: make a good name

**Review by manjilthapa — COMMENTED** [review]
okay

**Review by manjilthapa — APPROVED** [review]

**Pull request merged**